src/vm/gdbjit.h

```diff
@@ -140,7 +141,7 @@ class LocalsInfo
     NewArrayHolder< NewArrayHolder<char> > localsName;
     NewArrayHolder<Scope> localsScope;
     ULONG32 countVars;
-    ICorDebugInfo::NativeVarInfo *pVars;
+    NewArrayHolder<ICorDebugInfo::NativeVarInfo> pVars;
```

Comment:
A nit - now that this is no longer a pointer, could you please rename it to "vars" so that the name matches other array holders in this class? LGTM otherwise.